webdriver/tests/perform_actions/wheel.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+
+from webdriver.error import NoSuchWindowException
+
+from tests.perform_actions.support.refine import filter_dict, get_events
+from tests.support.asserts import assert_move_to_coordinates
+
+
+def test_null_response_value(session, wheel_chain):
+    value = wheel_chain.scroll(0, 0, 0, 10).perform()
+    assert value is None
+
+
+def test_no_browsing_context(session, closed_window, wheel_chain):
+    with pytest.raises(NoSuchWindowException):
+        wheel_chain.scroll(0, 0, 0, 10).perform()
+
+
+def test_wheel_scroll(session, test_actions_scroll_page, wheel_chain):
+    session.execute_script("document.scrollingElement.scrollTop = 0")
+
+    outer = session.find.css("#outer", all=False)
+    wheel_chain.scroll(0, 0, 5, 10, origin=outer).perform()
+    events = get_events(session)
+    assert len(events) > 0
```

Comment:
Do we expect to see a different number of events here? It would be good to have an equal check instead. This also applies to the other tests in this file.

Reply:
Yes, we do not know for sure how many events we will get because this test may get a mouse move which depends on the current mouse position.

Reply:
Please do so. That way we can assert for a clear number of wheel events.